packages/components/src/components/table/table-head-cell/table-head-cell.tsx

```diff
@@ -47,17 +52,23 @@ export class TableHeadCell {
   public render(): JSX.Element {
     validateProps(this, propTypes);
     const { active, direction } = this.sort || {};
-    attachComponentCss(this.host, getComponentCss, active, direction, this.hideLabel, this.multiline);
+    attachComponentCss(this.host, getComponentCss, active, direction, this.hideLabel, this.multiline, this.theme);
 
     const PrefixedTagNames = getPrefixedTagNames(this.host);
 
     return (
       <Host scope="col" role="columnheader" aria-sort={getAriaSort(this.sort)}>
         {isSortable(active, direction) ? (
-          <button type="button" onClick={this.onButtonClick}>
+          <PrefixedTagNames.pButtonPure class="button-pure" icon="none" onClick={this.onButtonClick} theme={this.theme}>
             <slot />
-            <PrefixedTagNames.pIcon class="icon" color="inherit" name="arrow-up" aria-hidden="true" />
-          </button>
+            <PrefixedTagNames.pIcon // we use slotted icon instead of built-in icon because we need to style it from here (for active state and for sorting)
```

Comment:
Why can't the same be achieved with the button?  There is an `active` prop and the spinning could be done on it as well.

Reply:
@denyo The behavior here is different from the "active" state of the button-pure: for active state the icon should be shown by default, and for "not-active" state icon should be shown only after hovering over button. See an example here: https://designsystem.porsche.com/issue/2147/components/table/examples#advanced-table (when hovering over "Purchase Intention" column for example)

So we need to style icon separately from button-pure:
<img width="687" alt="Bildschirm­foto 2023-03-08 um 09 03 26" src="https://user-images.githubusercontent.com/2981884/223656475-b5bc2649-b29d-4e61-a984-785fcb21887f.png">